yaml/csi-driver/edge/hpe-csi-k8s-1.29.yaml

```diff
@@ -812,7 +812,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-volume-group-provisioner
-          image: quay.io/hpestorage/volume-group-provisioner:v1.0.6
+          image: quay.io/hpestorage/volume-group-provisioner:v1.0.8-beta
```

Comment:
Is this version mentioned in the Dockerfile of the respective repos. I see nfs provisioner repo's Dockerfile updated with the new image version, but other repos are missing

Reply:
The next release tag is 1.0.7, but it was used by mistake sometime back. We deleted the 1.0.7 tag from the registry as it was not used anywhere, and now we are using the 1.0.7 tag. For side cards, the beta tag was added only for the images as there were not many changes.